lib/AdaptableFargateService.ts

```diff
@@ -48,14 +48,14 @@ export abstract class AdaptableConstruct extends Construct {
    * @returns A certificate value indicates ssl.
    */
   useSSL(): boolean {
-    return this.context.DNS.certificateARN ? true : false;
+    return this.context?.DNS?.certificateARN ? true : false;
   }
   
   /**
    * Get the vpc by checking for it in the properties supplied to the construct, else look it up using the
    * VPC_ID context value, resorting to creating a new vpc if either return no vpc.
    */
-  public getVpc(): Vpc {
+  public getVpc = (): Vpc => {
```

Comment:
Just curious, why make this an anonymous arrow function instead of a named function?

Reply:
Answer: You cannot properly deconstruct a function from it's parent object unless done this way.
This is not a concern for primities.

Reply:
Okay